src/components/Editor/SearchBar.js

```diff
@@ -399,12 +402,15 @@ const SearchBar = React.createClass({
   },
 
   // Handlers
-  selectResultItem(item: FormattedSymbolDeclaration) {
+  selectResultItem(e: SyntheticEvent, item: SymbolDeclaration, index?: number) {
```

Comment:
Once I only needed the item's location, `FormattedSymbolDeclaration` would expect more data than  I needed, and it could also break the _line length_ lint rule.